docs/infrastructure/index.asciidoc

```diff
@@ -15,15 +13,58 @@ image::infrastructure/images/infra-sysmon.jpg[Infrastructure Overview in Kibana]
 
 
 [float]
-== Add data sources
-Kibana provides step-by-step instructions to help you add your data sources.
-The {infra-guide}[Infrastructure Monitoring Guide] is a good source for more detailed
-instructions and information.
+== Add data
+
+Kibana provides step-by-step instructions to help you add log data. The
+{infra-guide}[Infrastructure Monitoring Guide] is a good source for more
+detailed information and instructions.
 
 [float]
 == Configure data sources 
 
-By default the Infrastructure UI uses the `metricbeat-*` index pattern to query the data. If you configured Metricbeat to export data to a different set of indices, you will need to set `xpack.infra.sources.default.metricAlias` in `config/kibana.yml` to match your index pattern. You can also configure the timestamp field by overriding `xpack.infra.sources.default.fields.timestamp`. See <<infrastructure-ui-settings-kb>> for a complete list.
+By default the Infrastructure UI uses the `metricbeat-*` index pattern to query
+the data.  If your metrics are located in a different set of indices or use a
+different timestamp field, you can adjust the source configuration via the user
+interface or the {kib} configuration file.
+
+NOTE: The Logs and Infrastructure UI share a common data source definition in
+each space. Changes in one of them can influence the data displayed in the
+other.
+
+[float]
+=== Configure data sources in the user interface
+
+The source configuration UI can be accessed via the corresponding button in the
+toolbar.
+
+[role="screenshot"]
+image::infrastructure/images/infrastructure-configure-source.png[Configure Infrastructure UI source button in Kibana]
+
+This opens the source configuration fly-out dialog, in which the following
+configuration items can be inspected and adjusted:
+
+* *Name*: The name of the source configuration.
+* *Indices*: The patterns of the elasticsearch indices to read metrics and logs
+  from.
+* *Fields*: The names of particular fields in the indices that need to be known
+  to the Infrastructure and Logs UIs in order to query and interpret the data
+  correctly.
+
+[role="screenshot"]
+image::infrastructure/images/infrastructure-configure-source-dialog.png[Configure Infrastructure UI source dialog in Kibana]
+
+TIP: If <<xpack-spaces>> are enabled in your Kibana instance, any configuration
+changes performed via the user interface are specific to that space. You can
```

Comment:
```suggestion
changes performed via source configuration in Kibana are specific to that space. You can
```

Reply:
Does that make it clear enough that this only applies to the source configuration via the UI and not via the config file?

Reply:
I think if we change the heading title from `Configure data sources in the user interface` to `Configure source` it is clear enough.